ui/src/app/components/runtime-configuration-panel/confirm-environment-delete-with-pd.tsx

```diff
@@ -1,29 +1,48 @@
 import * as React from 'react';
 
+import { Disk } from 'generated/fetch';
+
+import { UIAppType } from 'app/components/apps-panel/utils';
 import { Button } from 'app/components/buttons';
 import { FlexRow } from 'app/components/flex';
 import { ClrIcon } from 'app/components/icons';
 import { RadioButton } from 'app/components/inputs';
 import colors from 'app/styles/colors';
-import { ComputeType, detachableDiskPricePerMonth } from 'app/utils/machines';
+import { DEFAULT, switchCase } from 'app/utils';
+import { detachableDiskPricePerMonth } from 'app/utils/machines';
 import { formatUsd } from 'app/utils/numbers';
-import { RuntimeStatusRequest } from 'app/utils/runtime-utils';
 
 import { BackupFilesHelpSection } from './backup-files-help-section';
 import { styles } from './styles';
 
-const { useState, Fragment } = React;
+const { useState } = React;
+
+interface ConfirmDeleteEnvironmentWithPDProps {
```

Comment:
the naming issue was in the original, but let's rename the file to match this better

Reply:
I'm not entirely sure what you mean by this, but I fixed the mistake though and renamed `confirm-environment-delete-with-pd.tsx` to `confirm-delete-environment-with-pd.tsx` to match the component.

Reply:
Yep, that's what I meant